nncf/torch/dynamic_graph/wrappers.py

```diff
@@ -169,6 +170,30 @@ def wrapped(self, *args, **kwargs):
     return wrapped
 
 
+def _add_operation_attributes(node: 'DynamicGraphNode', ctx: TracingContext):
+    op_address = node.op_exec_context.op_address
+    # TODO(nlyalyus): support other operations: torch.addmm, torch.nn.functional.Linear, torch.bmm, torch.mm
+    if op_address.operator_name in ('matmul', 'bmm'):
+        graph = ctx.graph
+        input_edges = graph.get_input_edges(node)
+        # no input nodes when input_info tensor was bound to a *args or **kwargs on graph tracing
+        if len(input_edges):
+            assert len(input_edges) == 2
+            activation_edge, weight_edge = input_edges
+            activation_shape = activation_edge.activation_shape
```

Comment:
@ljaljushkin, can we use `args` and `kwargs` for operation attributes? For some operations (like `transpose`) layer attributes collection by input edges is impossible (`input_shape=(5,5,5), transpose=(2, 1, 0)`). You can find example with torch implementation for transpose layer attributes [here](https://github.com/daniil-lyakhov/nncf/commit/079578ccab88a18640ada7088b33f548e63937f4#diff-6e72f489b20539eb2b9d90608c21d9288a65638482687360312d8aebbc546871)